### PR TITLE
Datagrid: update the columns.name option

### PR DESCRIPTION
**Pull request by pavel-2020**

### CODE REVIEW

File: api-reference/_hidden/GridBaseColumn/name.md [hunk 1]
```diff
@@ -5,7 +5,8 @@ default: undefined
 ---
 ---
 ##### shortDescription
-Specifies the identifier of the column.
+Specifies a unique identifier of the column. If not set in code, the value is autogenerated from [dataField]({basewidgetpath}/Configuration/columns/#dataField).
```
Comment (RomanTsukanov) [inline api-reference/_hidden/GridBaseColumn/name.md]:
```suggestion
Specifies a unique identifier of the column. The value is inherited from the [dataField]({basewidgetpath}/Configuration/columns/#dataField) by default.
```

File: api-reference/_hidden/GridBaseColumn/name.md [hunk 2]
```diff
@@ -5,7 +5,8 @@ default: undefined
 ---
 ---
 ##### shortDescription
-Specifies the identifier of the column.
+Specifies a unique identifier of the column. The value is inherited from the [dataField]({basewidgetpath}/Configuration/columns/#dataField) by default.
```
Comment (dirk-pieterse) [inline api-reference/_hidden/GridBaseColumn/name.md]:
```suggestion
Specifies the column's unique identifier. This value is inherited from the [dataField]({basewidgetpath}/Configuration/columns/#dataField).
```

File: api-reference/_hidden/GridBaseColumn/name.md [hunk 3]
```diff
@@ -5,7 +5,8 @@ default: undefined
...
 
 ---
-Set this option if you need to refer to the column in your code afterwards, for example, when [changing a column option](/api-reference/10%20UI%20Widgets/GridBase/3%20Methods/columnOption(id_optionName_optionValue).md '{basewidgetpath}/Methods/#columnOptionid_optionName_optionValue').
\ No newline at end of file
+
+This option's value is used to refer to the column in your code, for example, when [changing a column option]({basewidgetpath}/Methods/#columnOptionid_optionName_optionValue).
```
Comment (dirk-pieterse) [inline api-reference/_hidden/GridBaseColumn/name.md]:
```suggestion
This option's value is used to refer to the column in code, for example, when [changing a column option]({basewidgetpath}/Methods/#columnOptionid_optionName_optionValue).
```

**Review by dirk-pieterse — APPROVED** [review]

File: api-reference/_hidden/GridBaseColumn/name.md (hunk 2 above)
Comment (pavel-2020) [reply api-reference/_hidden/GridBaseColumn/name.md]: This value is inherited from the [dataField] _only if the name option is not set in code_. Thus can the sentence be 

```suggestion
Specifies the column's unique identifier. If not set in code, this value is inherited from the [dataField]({basewidgetpath}/Configuration/columns/#dataField).
```

?

Comment (dirk-pieterse) [reply api-reference/_hidden/GridBaseColumn/name.md]: I agree with your suggestion, but your original version "The value is inherited from the [dataField]({basewidgetpath}/Configuration/columns/#dataField) by default" and "If not set in code, this value is inherited from the [dataField]({basewidgetpath}/Configuration/columns/#dataField)" is different.

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**